habitat/core/env.py

```diff
@@ -423,3 +424,238 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.close()
+
+
+class CLEnv(Env):
```

Comment:
Little iffy on the naming of CLEnv. It's not very discreptive. SequentialTaskEnv or something more descriptive would be helpful.

Reply:
Yes sorry about that, I narrowed down on context too much.